moseq2_extract/cli.py

```diff
@@ -248,10 +248,16 @@ def download_flip_file(config_file, output_dir):
 
 @cli.command(name="generate-config", help="Generates a configuration file (config.yaml) that holds editable options for extraction parameters.")
 @click.option('--output-file', '-o', type=click.Path(), default='config.yaml')
-def generate_config(output_file):
+@click.option('--azure', default=False, type=bool, help='specify if this is depth videos from Azure Kinect camera')
+
+def generate_config(output_file, azure):
 
     objs = extract.params
     params = {tmp.name: tmp.default for tmp in objs if not tmp.required}
+    if azure:
+        params['bg_roi_depth_range'] = [550, 650]
+        params['spatial_filter_size'] = [5]
+        params['tail_filter_size'] = [15, 15]
```

Comment:
As mentioned on the `moseq2-app` repo, `generate_config` should have a `camera` or `camera-type` parameter that defaults to `'k2'` instead. It is more flexible, and allows us to support other cameras in the future.

Reply:
Also, didn't you decide that we didn't need the spatial filter?

Reply:
we didn't need the spatial filter in PCA but decided to keep it in the extract part since the raw frame data is being saved and spatial filtering helps with locating the mouse. Lastly, spatical_filter_size is not used in the PCA step because the median spatial filtering parameter is called `medfilter-space`

Reply:
b5d88882dd4af1bb5196f48e3abe03664c575b76